sbi/inference/snle/snle_base.py

```diff
@@ -79,10 +79,7 @@ def __init__(
         self._summary.update({"mcmc_times": []})  # type: ignore
 
     def append_simulations(
-        self,
-        theta: Tensor,
-        x: Tensor,
-        from_round: int = 0,
+        self, theta: Tensor, x: Tensor, from_round: int = 0,
```

Comment:
black formatting is still off

Reply:
That's odd, I updated to the latest version. I am using black v 19.10b0

Reply:
When I run black on `snle_base.py` the file is left unchanged...

Reply:
Ok, I found the flaw. pip didn't let me update to the latest because of dependencies, now I have v21.